pkg/handler/handler.go

```diff
@@ -190,13 +194,30 @@ func (o *OpenAPIService) UpdateSpec(openapiSpec *spec.Swagger) (err error) {
 	return nil
 }
 
-func toProtoBinary(spec []byte) ([]byte, error) {
-	var info yaml.MapSlice
-	err := yaml.Unmarshal(spec, &info)
-	if err != nil {
-		return nil, err
+func jsonToYAML(j map[string]interface{}) yaml.MapSlice {
```

Comment:
does it matter if this loses the distinction between `nil` and `{}`?

Reply:
probably not. Fixed.